gems/delayed_job_active_record/4.1/active_record.rbs

```diff
@@ -0,0 +1,33 @@
+module Delayed
+  module Backend
+    module ActiveRecord
+      self.@configuration: Configuration
+
+      def self.configuration: -> Configuration
+      def self.configure: -> untyped
+
+      class Configuration
+        attr_reader reserve_sql_strategy: :optimized_sql
+        def initialize: -> void
+        def reserve_sql_strategy=: (Symbol val) -> Symbol
+      end
+
+      class Job
+        def self.set_delayed_job_table_name: -> String
+        def self.ready_to_run: (untyped worker_name, untyped max_run_time) -> untyped
+        def self.before_fork: -> untyped
+        def self.after_fork: -> untyped
+        def self.clear_locks!: (untyped worker_name) -> untyped
+        def self.reserve: (untyped worker, ?untyped max_run_time) -> nil
+        def self.reserve_with_scope: (untyped ready_scope, untyped worker, Time now) -> nil
+        def self.reserve_with_scope_using_optimized_sql: (untyped ready_scope, untyped worker, Time now) -> nil
```

Comment:
They, `reserve_with_scope_using_optimized_mysql` and `reserve_with_scope_using_optimized_mssql` probably return non-nil value. I guess `Job?` is the correct returned type, please confirm that.

Reply:
That is as you pointed out.
These methods return the value of `Job?`
We fixed them.